any_parser/any_parser.py

```diff
@@ -9,22 +9,19 @@
 
 import requests
 
-from any_parser.utils import (
-    ModelType,
-    check_file_type_and_path,
-    check_model,
-    upload_file_to_presigned_url,
-)
+from any_parser.utils import check_file_type_and_path, upload_file_to_presigned_url
 
 PUBLIC_SHARED_BASE_URL = "https://public-api.cambio-ai.com"
 TIMEOUT = 60
 
 
 class ProcessType(Enum):
-    FILE = "file"
-    TABLE = "table"
-    FILE_REFINED = "file_refined"
-    FILE_REFINED_QUICK = "file_refined_quick"
+    EXTRACT_PII = "extract_pii"
+    EXTRACT_TABLES = "extract_tables"
+    EXTRACT_KEY_VALUE = "extract_key_value"
+    EXTRACT_RESUME_KEY_VALUE = "extract_resume_key_value"
+    PARSE = "file"
```

Comment:
1. update this to parse.
2. update CDK to include parse / extract type. parse or extract to have its dedicated folder.
3. If type does not exist, in CDK raise exception.